crates/ruff_python_formatter/src/expression/parentheses.rs

```diff
@@ -71,6 +71,10 @@ pub enum Parentheses {
 
     /// Never add parentheses
     Never,
+
+    /// Switch call chain and attribute formatting to fluent style. This is otherwise identical to
+    /// `Never`, fluent style implies a set of outer parentheses
+    FluentStyle { outermost: bool },
```

Comment:
`FluentStyle` doesn't fit well into the `Parentheses` concept, which is intended to be generally applicable to all expressions.